onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -155,23 +155,23 @@ static Status GetExternalDataInfo(const ONNX_NAMESPACE::TensorProto& tensor_prot
 // This function does not unpack string_data of an initializer tensor
 static Status ReadExternalDataForTensor(const ONNX_NAMESPACE::TensorProto& tensor_proto,
                                         const ORTCHAR_T* tensor_proto_dir,
-                                        std::unique_ptr<unsigned char[]>& unpacked_tensor,
-                                        SafeInt<size_t>& tensor_byte_size) {
+                                        std::vector<uint8_t>& unpacked_tensor) {
```

Comment:
is it possible to use std::byte?

Reply:
Unfortunately, seems CUDA does not have std::byte support yet, revert back to use uint8+t